### PR TITLE
Adds benchmarks for gogojsonpb and mongodb's bson lib

### PR DESCRIPTION
**Pull request by sgtsquiggs**
We're doing a lot of jsonpb encoding and decoding and it is not fast. Wanted to benchmark it along with everything else.

Also added mongodb's bson lib since a lot of newer work is using this lib instead of mgo.

### CODE REVIEW

**Comment by sgtsquiggs** [comment]
I did not update the readme since my local machine is an Apple M1 Max.

**Comment by alecthomas** [comment]
Please update the README - it doesn't matter what the machine is, the benchmarks will all be relative to each other.

File: README.md [hunk 1]
```diff
@@ -73,137 +73,143 @@ type A struct {
 
 ## Results
 
-2021-06-21 Results with Go 1.16.5 linux/amd64 on an Intel(R) Core(TM) i7-3630QM CPU @ 2.40GHz
+2021-06-21 Results with Go 1.17.8 Darwin/arm64 on an Apple M1 Max
```
Comment (alecthomas) [inline README.md:76]: Date?

File: go.mod [hunk 2]
```diff
@@ -5,50 +5,47 @@ go 1.13
...
+	go.mongodb.org/mongo-driver v1.5.1
 	golang.org/x/sys v0.0.0-20210616094352-59db8d763f22 // indirect
-	golang.org/x/tools v0.1.3 // indirect
 	gopkg.in/linkedin/goavro.v1 v1.0.5
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22
 	zombiezen.com/go/capnproto2 v2.17.0+incompatible
 )
+
+replace (
+	github.com/itsmontoya/mum v0.5.6 => github.com/mojura/enkodo v0.5.6
```
Comment (alecthomas) [inline go.mod:49]: Mmm annoying :\

**Review by alecthomas — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (sgtsquiggs) [reply go.mod:49]: Yeah ran into problems - these libs renamed themselves. Updating to the latest caused other issues.

**Comment by alecthomas** [comment]
Thanks!

**Pull request merged**